soh/soh/OTRGlobals.cpp

```diff
@@ -1660,6 +1660,11 @@ extern "C" CustomMessageEntry Randomizer_GetNaviMessage() {
     return CustomMessageManager::Instance->RetrieveMessage(Randomizer::NaviRandoMessageTableID, naviTextId);
 }
 
+extern "C" CustomMessageEntry Randomizer_GetIceTrapMessage() {
+    u16 iceTrapTextId = rand() % NUM_ICE_TRAP_MESSAGES;
```

Comment:
i feel like just using `rand()` should be fine here, but i'd like to hear from @jbodner09 and @leggettc18

Reply:
Nah you need the modulo operator to effectively keep it in the valid range. Otherwise it will pick any number from 0 to RAND_MAX, which is dependant on the platform's C standard library implementation but in all cases is too big for the iceTrapTextIds.

Reply:
ah, that's not what i meant, i meant just calling `rand()` instead of using a different form or RNG. i know we need to modulo

Reply:
Oh right. Yeah we should probably get something a bit more unified for all of our RNG stuff but for now this is consistent with other things we have set up with RNG.

Reply:
I can't remember who it was anymore and I can't find back the conversation on it, but someone was planning to implement a more robust way of generating random numbers, and the advice back then was to use rand() until then so they can all be easily replaced once that's done.